src/constraint_system/ecc/mod.rs

```diff
@@ -23,6 +24,54 @@ impl Point {
     pub fn y(&self) -> &Variable {
         &self.y
     }
+
+    /// Returns an identity point
+    pub fn identity(composer: &mut StandardComposer) -> Point {
+        let one = composer.add_witness_to_circuit_description(Scalar::one());
+        Point {
+            x: composer.zero_var,
+            y: one,
+        }
+    }
+
+    /// Converts an AffinePoint into a constraint system Point
+    /// without constraining the values
+    pub fn from_private_affine(
+        composer: &mut StandardComposer,
+        affine: dusk_jubjub::AffinePoint,
+    ) -> Point {
+        let x = composer.add_input(affine.get_x());
+        let y = composer.add_input(affine.get_y());
+        Point { x, y }
+    }
+    /// Converts an AffinePoint into a constraint system Point
+    /// without constraining the values
+    pub fn from_public_affine(
+        composer: &mut StandardComposer,
+        affine: dusk_jubjub::AffinePoint,
+    ) -> Point {
+        let point = Point::from_private_affine(composer, affine);
+        composer.constrain_to_constant(point.x, Scalar::zero(), -affine.get_x());
+        composer.constrain_to_constant(point.y, Scalar::zero(), -affine.get_y());
+
+        point
+    }
+
+    /// Conditionally selects a Point based on an input bit
+    /// If:
+    ///     bit == 1 => self,
+    ///     bit == 0 => point_b,
+    pub fn conditional_select(
+        &self,
+        composer: &mut StandardComposer,
+        bit: Variable,
+        point_b: Point,
+    ) -> Point {
+        let x = composer.conditional_select(bit, *self.x(), *point_b.x());
+        let y = composer.conditional_select(bit, *self.y(), *point_b.y());
+
+        Point { x, y }
+    }
```

Comment:
Would make sense to move this to `plonk_gadgets` under an `ecc` module?

Reply:
Same as above, let's leave it here. It's already public so anyone that imports PLONK can use it